app/src/components/page/Header/index.tsx

```diff
@@ -67,12 +68,7 @@ export const HeaderComponent: React.FC<HeaderProps> = (props: HeaderProps) => {
           <Link to={URL_REACT_LOGIN}>{LOGIN}</Link>
         </Button>
       )}
-      <Button
-        shape="circle"
-        icon={<GlobalOutlined />}
-        className="bg-transparent border-0"
-        type="primary"
-      />
+      {ENABLE_LANGUAGE_SWITCHER && <LanguageSwitcher />}
```

Comment:
> Actually seems like you have to specify the the supported languages otherwise it will not have language options.

That means we should also include `REACT_APP_SUPPORTED_LANGUAGES` as part of the condition?

Reply:
hmmm,.. interesting, yeah makes sense.
though I kindof like the idea of allowing all languages by default better.
that way I can more accurately enforce the contracts presented by these env variables i.e. Supported_languages will not have anything to do whether the language switcher is shown, but only affects the languages options shown, @kelvin-muchiri  thoughts?

Reply:
@p-netm Ok...sure I currently see no harm in doing this

Reply:
@p-netm Maybe if think of it in terms of is it ok to enable languages that have not been authorized for a project in case this env is omitted? Is it ok for the users of that project to enjoy the fruits of having a myriad of languages to switch from yet they did not partake in the translation?

Reply:
I will say no. This is because most countries want language implementations done in a specific way. I have see case in OpenSRP where we have had to hide some languages because the client didn't want them for their instance.

Reply:
What would I need to do to enable the language switcher but only allow switching between 2 specific languages?  How would the config for that look like?

Reply:
@moshthepitt Could you say that in a different way?

Reply:
@kelvin-muchiri I am basically wondering how we think the config will look like for a case where I only want the language switcher to include some two specific languages out of say 10 that are available.

Reply:
@p-netm Base in @dubdabasoduba comment, what do you think should be the default if the `SUPPORTED_LANGUAGES` is not set? Display all options or none?

Reply:
reverting to none.